Configuration/Skimming/python/PDWG_BPHSkim_cff.py

```diff
@@ -0,0 +1,72 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.StandardSequences.Reconstruction_cff import *
+
+# muons with trigger info
+from MuonAnalysis.MuonAssociators.patMuonsWithTrigger_cff import *
```

Comment:
In addition, we should not introduce a dependency of MuonAnalysis into production. patMuonsWithTriggerSequence should probably just be moved into PhysicsTools